composer.json

```diff
@@ -47,8 +47,8 @@
         "phpunit/phpunit": "^10.1"
     },
     "suggest": {
-        "ext-pcov": "*",
-        "ext-xdebug": "*"
+        "ext-pcov": "A fast code coverage driver",
+        "ext-xdebug": "Code coverage driver"
```

Comment:
A better description would be "PHP extension that provides line coverage as well as branch and path coverage".

Reply:
Thanks, I updated both descriptions